src/views/Login.vue

```diff
@@ -1,15 +1,22 @@
 <template>
   <v-row align="center" justify="center">
-      <v-card loading width="500">
+      <v-card :loading="$route.query.callback" width="500">
         <v-card-title>
-          Login
+          Freeflow Connect
```

Comment:
should the filename / routename be changed ?

Reply:
Dont think so